README.md

```diff
@@ -1 +1,14 @@
 # MediaMonks - Frontend Coding Standards
+
+## CSS
+
+We use [Sass](https://sass-lang.com/) as css preprocessor.
+
+#### Selectors
+Only use class names for styling, no id's.
+
+#### z-index
+Never use a `z-index` value higher then `4`. If you need more, use the [zindex](https://github.com/mediamonks/seng-scss/blob/master/utils/function/_zindex.scss) function of [seng-scss](https://www.npmjs.com/package/seng-scss).
+
+#### Vendor prefixes
+Never add vendor prefixes in the scss, use [autoprefixer](https://github.com/postcss/autoprefixer) to add them automatically. 
```

Comment:
There are vendor-unique styles that need a prefix as far as I know. I would replace it with something like:

Only use vendor-prefixes when they are unique for that vendor. [Autoprefixer](https://github.com/postcss/autoprefixer) will handle the vendor prefixes for non-unique properties like box-shadow if they are required for browsers in scope.

Reply:
I would strongly discourage entering vendor prefixes at all. Do we have compelling examples for vendor prefixes that cannot be handled with autoprefixer? I'd prefer those to be exceptions rather than a suggested course of action.

Reply:
Some properties are exclusive to certain browsers. For instance `-webkit-overflow-scrolling`

Reply:
That's one very popular (and also very non-standard) example. It looks like work is under way to cover that in [overscroll-behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior), so hopefully that won't be too long.

Instead, it would be better to not allow vendor prefixes in general but whitelist the compelling example in the linter. (See [value-no-vendor-prefix](https://stylelint.io/user-guide/rules/value-no-vendor-prefix#value-no-vendor-prefix) )

Reply:
Another one is `-webkit-clamp`, which is understood just fine in most modern browsers with just `-webkit-` prefix for now. ([link](https://css-tricks.com/almanac/properties/l/line-clamp/)) - this too is being [standardised](https://www.w3.org/TR/css-overflow-3/#line-clamp) but not yet available.